### ISSUE TITLE
Elements inside a flexbox doesn’t stretch when the align-self and align-items is set to stretch.

### ISSUE THREAD

**Issue by kmleb** [elementor/elementor#17052]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by kmleb** [comment]
The elements inside the flexbox container do not **stretch** even if the property set to **Align Self**    
<img width="1435" alt="Screenshot 2021-11-26 at 7 02 48 PM" src="https://user-images.githubusercontent.com/93880522/143617969-4a5e2bd1-6806-4a5e-a495-b3d0e5a60c7c.png">
.

**Comment by kmleb** [comment]
Hi @shilo-ey , Yes exactly. The background doesn't stretch.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.6.8

Feel free to update

Thanks!

**Comment by kmleb** [comment]
Great!
